From 7348794ded68cb2066e4906d9246ed8feb0d6109 Mon Sep 17 00:00:00 2001
From: Sascha Weisenberger <sascha.weisenberger@siemens.com>
Date: Wed, 6 Jul 2016 15:27:20 +0100
Subject: [PATCH 01/10] stmmac: adding support for platform IOT2000

IOT2000 covers both, IOT2020 and IOT2040, while only the IOT2040 has
two LAN ports.

Signed-off-by: Sascha Weisenberger <sascha.weisenberger@siemens.com>
---
 drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
index d71a721..07e24ef 100644
--- a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
+++ b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
@@ -138,6 +138,16 @@ static struct stmmac_pci_dmi_data quark_pci_dmi_data[] = {
 		.func = 6,
 		.phy_addr = 1,
 	},
+	{
+		.name = "SIMATIC IOT2000",
+		.func = 6,
+		.phy_addr = 1,
+	},
+	{
+		.name = "SIMATIC IOT2000",
+		.func = 7,
+		.phy_addr = 1,
+	},
 	{}
 };
 
-- 
2.10.2

